src/PluginsSystem/Microsoft.Performance.Toolkit.Plugins.Runtime/Package/PackageConstants.cs

```diff
@@ -8,7 +8,8 @@ namespace Microsoft.Performance.Toolkit.Plugins.Runtime.Package
     /// </summary>
     public static class PackageConstants
     {
-        public const string PluginMetadataFileName = "pluginspec.json";
+        public const string PluginMetadataFileName = "ptix.metadata.json";
```

Comment:
@trgibeau Any thoughts about the metadata file names?

Reply:
Are these the ones packed into the ptix package?

if so, we probably can just have metadata.json & contentsmetadata.json

Reply:
Yes

Reply:
Updated